compiler/src/iree/compiler/Codegen/LLVMGPU/LLVMGPUTileTensor.cpp

```diff
@@ -14,6 +14,7 @@
 #include "iree/compiler/Codegen/Transforms/Transforms.h"
 #include "iree/compiler/Codegen/Utils/GPUUtils.h"
 #include "iree/compiler/Codegen/Utils/MarkerUtils.h"
+#include "llvm/Support/Debug.h"
```

Comment:
Is this needed?

Reply:
Unfortunately it is, the include used to be in the .h and was pulled transitively everywhere.
I cleaned that up as I was at it and this is the consequence.